src/app/app.component.ts

```diff
@@ -1,10 +1,87 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Item } from './models/item';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'angular-green-grocers';
-}
+  cart: { item: Item, quantity: number }[] = [];
+  items: Item[] = [
```

Comment:
You shouldn't be hard-coding the items. You wouldn't be able to do this in a live project.